lib/actions/device.coffee

```diff
@@ -126,7 +126,9 @@ exports.remove =
 
 			(confirmed, callback) ->
 				return callback() if not confirmed
-				resin.models.device.remove(params.uuid).nodeify(callback)
+				resin.models.device.get(params.uuid).then (device) ->
```

Comment:
Since we're not using the result of the `.get()`, function, can you omit the `(device)` argument?